crypto/kyber/pqcrystals_kyber_ref_common/kem.c

```diff
@@ -52,8 +52,8 @@ int crypto_kem_keypair(uint8_t *pk,
                        uint8_t *sk)
 {
   uint8_t coins[2*KYBER_SYMBYTES];
-  pq_custom_randombytes(coins, KYBER_SYMBYTES);
-  pq_custom_randombytes(coins + KYBER_SYMBYTES, KYBER_SYMBYTES);
+  RAND_bytes(coins, KYBER_SYMBYTES);
```

Comment:
Why this change? Now isn't it impossible to test the randomized version of Kyber? Also this concern would be fixed if we just removed this randomized version and handled this in the EVP layer.

Reply:
The implementation of kyber and ML-KEM are such that the de-randomized versions contain the algorithm, and the randomized versions simply wrap the de-randomized versions with a call to obtain randomness on behalf of the user. As such, we are always hitting the core kyber/ML-KEM "randomized" code.

See the wrapping here:

https://github.com/jakemas/aws-lc/blob/35097c28bb96f1b29d064250669682203a762eb2/crypto/kyber/pqcrystals_kyber_ref_common/kem.c#L11-L59